machines/openId4VP/openId4VPActions.ts

```diff
@@ -0,0 +1,207 @@
+import {assign} from 'xstate';
+import {send, sendParent} from 'xstate/lib/actions';
+import {SHOW_FACE_AUTH_CONSENT_SHARE_FLOW} from '../../shared/constants';
+import {VC} from '../VerifiableCredential/VCMetaMachine/vc';
+import {StoreEvents} from '../store';
+
+import {VCShareFlowType} from '../../shared/Utils';
+
+let predefinedPresentationDefinitions = {
+  sunbird_rc_insurance_vc_ldp: {
+    id: 'vp token example',
+    purpose:
+      'Philippines Govt is requesting your digital ID for the purpose of Self-Authentication',
```

Comment:
don't refer to any entity, use generic purpose.
something like `Relying party is requesting your digital ID for the purpose of Self-Authentication`